### DISCUSSION TITLE
More translation strings

### DISCUSSION THREAD

**Discussion by Blueberryy** [night/betterttv#5838] [Ideas]
Can you add these to website translation file? 
![image](https://user-images.githubusercontent.com/36592509/211289155-182dc4be-86bf-49f3-ab6a-2b318ac0dfe6.png)

Also badge names and extension page descriptions on Firefox and Chrome would be awesome to translate

**Comment by night** [comment]
Thanks for the suggestion. We're still working through all the remaining translation strings, but there will likely be a long tail of strings we may miss. I'll follow-up here once we've shipped the majority of translation strings.

**Comment by night** [comment]
Version 7.5.1 which is in the process of releasing contains the remainder of strings. There are definitely still gaps in coverage but we may want to tackle those on an individual basis as issues rather than feature requests.
